polaris-core/src/main/java/io/polaris/core/persistence/LocalPolarisMetaStoreManagerFactory.java

```diff
@@ -51,7 +51,7 @@ public abstract class LocalPolarisMetaStoreManagerFactory<StoreType>
 
   protected PolarisStorageIntegrationProvider storageIntegration;
 
-  private Logger logger =
+  private Logger LOGGER =
```

Comment:
```suggestion
  private static final Logger LOGGER =
```

Reply:
The following classes looks missing `static`:
* AzureCredentialStorageIntegrationTest 
* ManifestFileCleanupTaskHandler
* TracingFilter

Also, we should make the constant to `private` in the following classes:
* DefaultOAuth2ApiService
* PolarisApplicationIntegrationTest